syft/format/internal/spdxutil/helpers/download_location.go

```diff
@@ -17,17 +36,17 @@ func DownloadLocation(p pkg.Package) string {
 	if hasMetadata(p) {
 		switch metadata := p.Metadata.(type) {
 		case pkg.ApkDBEntry:
-			return NoneIfEmpty(metadata.URL)
+			return checkUri(metadata.URL)
```

Comment:
Instead of updating every spot here to call `checkUri`, could we centralize this call? maybe add a var `location := ""`, set this variable in each of these case statements instead of returning, and `return UriValue(location)`

Reply:
I've updated it, but it does lead to some change in one of the testcases, where providing no package result in NONE instead of NOASSERTION, which makes more sense in my opinion anyway.

Reply:
Agree -- NONE seems like the right value